src/apps/CMakeLists.txt

```diff
@@ -6,14 +6,16 @@ if(OCIO_BUILD_APPS)
 	add_subdirectory(ociocheck)
 	add_subdirectory(ociochecklut)
 	add_subdirectory(ociomakeclf)
+	add_subdirectory(ocioperf)
 	add_subdirectory(ociowrite)
 
 	if(OpenImageIO_FOUND AND TARGET OpenImageIO::OpenImageIO)
 		add_subdirectory(ociolutimage)
 		add_subdirectory(ocioconvert)
 		add_subdirectory(ociodisplay)
-		add_subdirectory(ocioperf)
```

Comment:
I think this should be put back in because it relies on an earlier PR #1516 that is not in RB-2.0 and probably don't need to be. Looking at the commit you used mentioned in the description, it's not clear to me how it got here.

Reply:
I think that I know what happened. I will rebase off this PR from RB-2.1 anyway.

Reply:
Ah ok, and yeah just realised I assumed it was targeting 2.0 but it's actually 2.1!

Reply:
Sounds good, I will rename the PR title! I have just rebased everything from RB-2.1, it should be much cleaner now.

Reply:
The test suite is seemingly passing now, good sign!